frontend_vue/src/components/constants.ts

```diff
@@ -66,3 +67,21 @@ export const INCIDENT_LIST_EXPORT = {
   JSON: 'incidentlist_json',
   CSV: 'incidentlist_csv',
 };
+
+export const INCIDENT_CHANNEL_TYPE_LABELS = {
+  aws_keys: 'AWS API Key Token',
+  cc: 'Credit Card Token',
+  azure_id: 'Azure Login Certificate Token',
+};
```

Comment:
this seems wrong to me. these aren't input channels. the input channels are:

```
INPUT_CHANNEL_HTTP = "HTTP"
INPUT_CHANNEL_DNS = "DNS"
INPUT_CHANNEL_IMGUR = "Imgur"
INPUT_CHANNEL_LINKEDIN = "LinkedIn"
INPUT_CHANNEL_BITCOIN = "Bitcoin"
INPUT_CHANNEL_SMTP = "SMTP"
INPUT_CHANNEL_MTLS = "Kubeconfig"
INPUT_CHANNEL_MYSQL = "MYSQL"
INPUT_CHANNEL_WIREGUARD = "WireGuard"
```

These seem more like token type labels?

Reply:
That's what I found in the current forntend setup:
`  {% if canarydrop['type'] == 'aws_keys' %}
                                  <b>Channel:</b> AWS API Key Token
  {% elif canarydrop['type'] == 'cc' %}
                                  <b>Channel:</b> Credit Card Token
  {% elif canarydrop['type'] == 'azure_id' %}
                                  <b>Channel:</b> Azure Login Certificate Token
  {% else %}
                                  <b>Channel:</b> {{ canarydrop['triggered_list'][item]['input_channel'] }}
  {% endif %}`
  
  It looks like we want to display a custom channel label for these specific token types.
  If these variables exist on the backend, we can maybe implement them for these token types as well. Instead of adding this channel check on the frontend side

Reply:
what do the other tokens display for Channel on old tokens.org?

Reply:
heh, that would be good to know. I'm not sure how to test most of the tokens, all I can rely on is the current history file

Reply:
So here's an example. Is this the correct place we're using it? How are we handling the DNS channel? Looking at your code it looks like we aren't?

![Screenshot 2024-05-08 at 13 51 11](https://github.com/thinkst/canarytokens/assets/25585356/1a2e936a-78b3-497a-a94a-272ef9e5558d)